website/docs/r/machine_learning_inference_cluster.html.markdown

```diff
@@ -130,10 +130,19 @@ The following arguments are supported:
 
 * `description` - (Optional) The description of the Machine Learning compute. Changing this forces a new Machine Learning Inference Cluster to be created.
```

Comment:
Whenever we speak about "Machine Learning Inference Cluster", could we say "Azure Kubernetes Service cluster" instead? Azure Machine Learning merely helps create an Azure Kubernetes Service cluster programmatically as described on this page: https://docs.microsoft.com/en-us/azure/machine-learning/how-to-create-attach-kubernetes?tabs=python. The name `Machine Learning Inference Cluster` may create the impression that Inference Cluster is a managed offering although it is not.

Reply:
I see. But "Azure Kubernetes Service cluster" sounds like https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/kubernetes_cluster

Reply:
We actually are deploying https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/kubernetes_cluster it, and then register it to the machine learning workspace. Customers can also register an existing AKS cluster. To our end users, its the same thing.

Reply:
This is the description assigned to this resource? and this resource is a `Machine Learning Inference Cluster` regardless of what it is behind the scenes.

Reply:
@katbyte We never expected `Machine Learning Inference Cluster` by name as its concept is not existing in our service.